applications/utils.py

```diff
@@ -13,6 +13,6 @@ def validate_file_extension(value):
         value (File): file name
     """
     ext = os.path.splitext(value.name)[1]
-    valid_extensions = [".pdf", ".doc", ".docx", "odt"]
+    valid_extensions = [".pdf", ".doc", ".docx", ".odt"]
```

Comment:
this is not related to this PR, and is not going to cause any problem right now. I do not want to touch it because it is not a single line change and it involves tests.